SPECS/LICENSES-AND-NOTICES/data/licenses.json

```
@@ -2355,6 +2356,8 @@
                 "hawtjni-runtime",
                 "httpcomponents-client",
                 "httpcomponents-core",
+                "influxdb2",
+                "influx-cli",
```

Please include entries in alphabetical order.
```suggestion
                "influx-cli",
                "influxdb2",
```

Btw. this is done for you automatically if you run this command from the repo's root:
```bash
toolkit/scripts/license_map.py --no_check --update --remove_missing SPECS/LICENSES-AND-NOTICES/data/licenses.json SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md SPECS SPECS-EXTENDED SPECS-SIGNED
```

---

I tried to run that as one of the checks was suggesting it, and it did a lot of changes, mainly re-arranging a lot of licenses